### PR TITLE
Travis and pre release

### PR DESCRIPTION
**Pull request by hjanuschka**
* add travis
  * auto release a new `-pre` version on every on `master`

you have to run: 

```bash
travis encrypt --add deploy.api_key
```

and copy&paste your rubygems api key.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,11 @@
-# os: osx # enable this if you need macOS support
 language: ruby
 rvm:
-  - 2.2.4
```
Comment (hjanuschka) [inline .travis.yml:4]: the changes here are from rubocop

File: fastlane-plugin-docker.gemspec [hunk 2]
```diff
@@ -6,10 +6,11 @@ require 'fastlane/plugin/docker/version'
...
   spec.name          = 'fastlane-plugin-docker'
   spec.version       = Fastlane::Docker::VERSION
-  spec.author        = %q{Manu Wallner}
-  spec.email         = %q{manu@supermil.ch}
+  spec.version       = "#{spec.version}-alpha-#{ENV['TRAVIS_BUILD_NUMBER']}" if ENV['TRAVIS']
+  spec.author        = 'Manu Wallner'
+  spec.email         = 'manu@supermil.ch'
 
-  spec.summary       = %q{fastlane Actions to support building images, logging into Docker Hub, and pushing those images to the Hub}
+  spec.summary       = 'fastlane Actions to support building images, logging into Docker Hub, and pushing those images to the Hub'
```
Comment (hjanuschka) [inline fastlane-plugin-docker.gemspec:13]: changes are from rubocop

**Comment by milch** [comment]
Pretty cool, thanks! 🚀

**Review by milch — APPROVED** [review]
👍

**Pull request merged**